subscription_oca/views/sale_subscription_views.xml

```diff
@@ -370,12 +370,67 @@
         <field name="model">sale.subscription</field>
         <field name="arch" type="xml">
             <search>
+                <field
+                    name="name"
+                    filter_domain="['|', ('name','ilike',self), ('code','ilike',self)]"
+                />
                 <field name="to_renew" />
+                <field name="description" />
                 <filter
                     string="Pending subscriptions"
                     name="pendingsubs"
                     domain="[('to_renew','=', True)]"
                 />
+                <field name="sale_order_id" />
+                <field name="journal_id" />
+                <field name="pricelist_id" />
+                <separator />
+                <separator />
```

Comment:
**question**: Is there a reason for the two separators in a row? They don't seem to show up in the interface

Reply:
The reason is the Issue asked to use the same filters structure of those for `contract.contract` in `sale.subscription`

Reply:
But I can change these, no problem